### PR TITLE
Tctl disperse info

### PR DESCRIPTION
**Pull request by DJSanti**
reviewing how-to-use-tctl for duplicate information and outdated things. whatever can be moved will be moved to approprriate files. I hope to delete this thing.

### CODE REVIEW

File: docs/tctl/how-to-use-tctl.md [hunk 1]
```diff
@@ -262,294 +120,54 @@ Otherwise your workflow can't make any progress after auto-reset.
...
-```
-
-_ActivityTaskStarted_ event is recorded when the Activity task is picked up by an Activity worker.
-The Identity field contains the Id of the worker (you can set it to any value on worker startup).
-
-_ActivityTaskCompleted_ event is recorded when Activity completes.
-It contains the result of the Activity execution.
+:::note
+garbage
+:::
```
[the remark this answers is not in the document]
Comment (DJSanti) [reply docs/tctl/how-to-use-tctl.md]: this file has been deleted btw. Garbage cleanup was the goal. Those were just my placeholders

File: docs/tctl/namespace/update.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ tctl namespace update --active_cluster <name>
 ### `--add_bad_binary`
 
 Add a binary checksum to use when resetting a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+Temporal will not dispatch any command tasks to the given binary.
```
Comment (flossypurse) [inline docs/tctl/namespace/update.md:32]: What is a command task here?

File: docs/tctl/workflow/run.md [hunk 3]
```diff
@@ -104,7 +104,11 @@ tctl workflow run --cron <string>
 Specify a [Workflow Id Reuse Policy](/concepts/what-is-a-workflow-id-reuse-policy).
 Configure if the same [Workflow Id](/concepts/what-is-a-workflow-id) is allowed for use in new [Workflow Execution](/concepts/what-is-a-workflow-execution).
 
-Values: `AllowDuplicate`, `AllowDuplicateFailedOnly`, `RejectDuplicate`
+There are three allowed values:
+
+- **AllowDuplicateFailedOnly:** Allow starting a Workflow execution using the same Workflow Id when a Workflow with the same Workflow Id is not already running and the last execution close state is one of _[terminated, cancelled, timedout, failed]_.
```
Comment (flossypurse) [inline docs/tctl/workflow/run.md:109]: We can link to the definition of these -

File: docs/tctl/workflow/start.md [hunk 4]
```diff
@@ -92,7 +104,11 @@ tctl workflow start --cron <string>
 Specify a [Workflow Id Reuse Policy](/concepts/what-is-a-workflow-id-reuse-policy).
 Configure if the same [Workflow Id](/concepts/what-is-a-workflow-id) is allowed for use in new [Workflow Execution](/concepts/what-is-a-workflow-execution).
 
-Values: `AllowDuplicate`, `AllowDuplicateFailedOnly`, `RejectDuplicate`
+There are three allowed values:
+
+- **AllowDuplicateFailedOnly:** Allow starting a Workflow execution using the same Workflow Id when a Workflow with the same Workflow Id is not already running and the last execution close state is one of _[terminated, cancelled, timedout, failed]_.
```
Comment (flossypurse) [inline docs/tctl/workflow/start.md:109]: can link to the definition here too.

File: docs/tctl/workflow/run.md (hunk 3 above)
Comment (DJSanti) [reply docs/tctl/workflow/run.md:109]: Other than the enum page and others, what definition is there to link? There isn't a set thing; these definitions are repeated across pages.

File: docs/tctl/workflow/describe.md [hunk 5]
```diff
@@ -9,17 +9,21 @@ tags:
 ---
 
 The `tctl workflow describe` command shows information about a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+This information can be used to locate any deployment where a Workflow failed.
```
Comment (flossypurse) [inline docs/tctl/workflow/describe.md:12]: What is a "deployment" here?
"... to locate any Workflow Execution that might have failed, for example"

File: docs/tctl/workflow/describe.md [hunk 6]
```diff
@@ -58,6 +62,7 @@ tctl workflow describe --run_id <id>
 ### `--reset_points_only`
 
 Show only events that are eligible for reset.
+If successful, the command returns the Run Id of all deployments, and the times at which the Event Ids were created.
```
Comment (flossypurse) [inline docs/tctl/workflow/describe.md:65]:
```suggestion
If successful, the command returns the Run Id of all deployments, and the times at which the Event Ids were created.
```

Event Ids or .. just Events?

File: docs/tctl/workflow/reset-batch.md [hunk 7]
```diff
@@ -8,11 +8,13 @@ tags:
   - tctl
 ---
 
-The `tctl workflow reset-batch` command resets a batch of [Workflow Executions](/concepts/what-is-a-workflow-execution) by `resetType`.
+The `tctl workflow reset-batch` command resets a batch of [Workflow Executions](/concepts/what-is-a-workflow-execution) by [`resetType`](#resettype).
 
-See also [`tctl workflow reset`](/tctl/workflow/reset).
+Resetting a Workflow allows the process to be resumed from a certain point without losing your parameters or Work History.
```
Comment (flossypurse) [inline docs/tctl/workflow/reset-batch.md:13]:
```suggestion
Resetting a Workflow allows the process to be resumed from a certain point without losing your parameters or Event History.
```

File: docs/tctl/workflow/reset.md [hunk 8]
```diff
@@ -8,11 +8,13 @@ tags:
   - tctl
 ---
 
-The `tctl workflow reset` command resets a [Workflow Execution](/concepts/what-is-a-workflow-execution) by either `eventId` or `resetType`.
+The `tctl workflow reset` command resets a [Workflow Execution](/concepts/what-is-a-workflow-execution) by either [`eventId`](#eventid)or [`resetType`](#resettype).
 
-See also [`tctl workflow reset-batch`](/tctl/workflow/reset-batch).
+Resetting a Workflow allows the process to be resumed from a certain point without losing your parameters or Work History.
```
Comment (flossypurse) [inline docs/tctl/workflow/reset.md:13]:
```suggestion
Resetting a Workflow allows the process to be resumed from a certain point without losing your parameters or Event History.
```

File: docs/tctl/workflow/reset.md [hunk 9]
```diff
@@ -8,11 +8,13 @@ tags:
...
 
-`tctl workflow reset [<modifiers>]`
+To run multiple Reset operations at once, see [`tctl workflow reset-batch`](/tctl/workflow/reset-batch).
```
Comment (flossypurse) [inline docs/tctl/workflow/reset.md:15]: I was going to say "Reset is not a core term, let's not capitalize..." - but I am actually thinking maybe it is...
Might want to make a note in the Docs project plan to explain this term -

File: docs/tctl/workflow/run.md (hunk 3 above)
Comment (flossypurse) [reply docs/tctl/workflow/run.md:109]: They are defined here: https://docs.temporal.io/workflows#workflow-id-reuse-policy

I think we should turn each of them into a header and you can list them, but link to each one directly - or something.
I usually prefer to link to source than repeat that many times.

File: docs/tctl/workflow/describe.md (hunk 5 above)
Comment (flossypurse) [inline docs/tctl/workflow/describe.md:12]:
```suggestion
This information can be used to locate a failed Workflow Execution, for example.
```

File: docs/tctl/namespace/index.md [hunk 10]
```diff
@@ -7,10 +7,10 @@ tags:
   - tctl
 ---
 
-The `tctl namespace` commands enable [Namespace](/concepts/what-is-a-namespace) operations.
-
 Alias: `n`
 
+The `tctl namespace` commands enable [Namespace](/concepts/what-is-a-namespace) operations.
```
Comment (flossypurse) [inline docs/tctl/namespace/index.md:12]: Let's keep the first sentence description above the Alias

**Review by flossypurse — APPROVED** [review]

**Pull request merged**